src/Module/Admin/Site.php

```diff
@@ -194,131 +194,133 @@ protected function post(array $request = [])
 				);
 			}
 		}
-		DI::config()->set('system', 'ssl_policy'            , $ssl_policy);
-		DI::config()->set('system', 'maxloadavg'            , $maxloadavg);
-		DI::config()->set('system', 'min_memory'            , $min_memory);
-		DI::config()->set('system', 'optimize_tables'       , $optimize_tables);
-		DI::config()->set('system', 'contact_discovery'     , $contact_discovery);
-		DI::config()->set('system', 'synchronize_directory' , $synchronize_directory);
-		DI::config()->set('system', 'poco_requery_days'     , $poco_requery_days);
-		DI::config()->set('system', 'poco_discovery'        , $poco_discovery);
-		DI::config()->set('system', 'poco_local_search'     , $poco_local_search);
-		DI::config()->set('system', 'nodeinfo'              , $nodeinfo);
-		DI::config()->set('config', 'sitename'              , $sitename);
-		DI::config()->set('config', 'sender_email'          , $sender_email);
-		DI::config()->set('system', 'suppress_tags'         , $suppress_tags);
-		DI::config()->set('system', 'shortcut_icon'         , $shortcut_icon);
-		DI::config()->set('system', 'touch_icon'            , $touch_icon);
+		DI::config()->set('system', 'ssl_policy'            , $ssl_policy, false);
+		DI::config()->set('system', 'maxloadavg'            , $maxloadavg, false);
+		DI::config()->set('system', 'min_memory'            , $min_memory, false);
+		DI::config()->set('system', 'optimize_tables'       , $optimize_tables, false);
+		DI::config()->set('system', 'contact_discovery'     , $contact_discovery, false);
+		DI::config()->set('system', 'synchronize_directory' , $synchronize_directory, false);
+		DI::config()->set('system', 'poco_requery_days'     , $poco_requery_days, false);
+		DI::config()->set('system', 'poco_discovery'        , $poco_discovery, false);
+		DI::config()->set('system', 'poco_local_search'     , $poco_local_search, false);
+		DI::config()->set('system', 'nodeinfo'              , $nodeinfo, false);
+		DI::config()->set('config', 'sitename'              , $sitename, false);
+		DI::config()->set('config', 'sender_email'          , $sender_email, false);
+		DI::config()->set('system', 'suppress_tags'         , $suppress_tags, false);
+		DI::config()->set('system', 'shortcut_icon'         , $shortcut_icon, false);
+		DI::config()->set('system', 'touch_icon'            , $touch_icon, false);
 
 		if ($banner == "") {
-			DI::config()->delete('system', 'banner');
+			DI::config()->set('system', 'banner', false);
```

Comment:
Without the 4th argument this would immediately save the config file, is this intended?